CIScripts/BuildStage.ps1

```diff
@@ -3,9 +3,10 @@
 $OutputRootDirectory = "output"
 $Password = $Env:WINCIDEV_PSW | ConvertTo-SecureString -asPlainText -Force
 $Credentials = New-Object System.Management.Automation.PSCredential ($Env:WINCIDEV_USR, $Password)
+$CopyDisabledArtifacts = Test-Path Env:COPY_DISABLED_ARTIFACTS
```

Comment:
This is kinda outside the scope of this PR but if you like, maybe you could import Aliases.ps1 and use PSCredentialT instead of System.Management.Automation.PSCredential?